src/main/scala/ai/privado/auth/AuthenticationHandler.scala

```diff
@@ -69,6 +70,7 @@ object AuthenticationHandler {
 
   def askForPermission(): Boolean = {
     println("Do you want to visualize these results on our Privacy View Cloud Dashboard? (Y/n)")
+    println("Once consent is given you can use -su, or --skip-upload flag with 'scan' command to skip the upload")
```

Comment:
@pandurangpatil Can we change this message?
Privado-CLI will not add `-su` command, as multicharacter options without `--` are not allowed, hence only `--skip-upload` will be present.

Also, since this message will be visible to the user after the question, and `(Y/n)` prompt will slide up, will it still be intuitive for user to press (Y/n) to answer the question? Worth thinking, you can decide.

Reply:
Will check with a wider audience.